consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/IndividualRequestConsumerTest.java

```diff
@@ -120,115 +128,85 @@ public class IndividualRequestConsumerTest extends AbstractSdtUnitTestBase {
         individualRequestConsumer.setRethrowOnFailureToConnect(true);
 
         individualRequest = this.createIndividualRequest();
         individualRequestType = this.createRequestType(individualRequest);
+    }
 
+    @Test
+    void getClient() {
+        final String targetApplicationCode = "";
+        final String serviceType = "";
+        final String webServiceEndPoint = "";
+        final long connectionTimeOut = 0L;
+        final long receiveTimeOut = 0L;
+        ITargetAppInternalEndpointPortType portType = individualRequestConsumer.getClient(targetApplicationCode,
+                serviceType, webServiceEndPoint, connectionTimeOut, receiveTimeOut);
+        Assertions.assertTrue(null != portType);
     }
 
     /**
      * Test method for successful processing of individual request.
      */
     @Test
-    public void processIndividualRequestSuccess() {
+    void processIndividualRequestSuccess() {
         final IndividualResponseType individualResponseType = generateResponse();
 
-        EasyMock.expect(mockTransformer.transformDomainToJaxb(individualRequest)).andReturn(individualRequestType);
+        when(mockTransformer.transformDomainToJaxb(individualRequest)).thenReturn(individualRequestType);
 
-        EasyMock.expect(mockClient.submitIndividual(individualRequestType)).andReturn(individualResponseType);
+        when(mockClient.submitIndividual(individualRequestType)).thenReturn(individualResponseType);
 
         mockTransformer.transformJaxbToDomain(individualResponseType, individualRequest);
 
-        EasyMock.expectLastCall().andAnswer(new IAnswer<Object>() {
-            @Override
-            public Object answer() throws Throwable {
-                ((IndividualRequest) EasyMock.getCurrentArguments()[1])
-                        .setRequestStatus(IIndividualRequest.IndividualRequestStatus.ACCEPTED.getStatus());
-                // required to be null for a void method
-                return null;
-            }
-        });
+        doAnswer((Answer<Void>) invocation -> {
+            ((IndividualRequest) invocation.getArgument(0))
+                    .setRequestStatus (IIndividualRequest.IndividualRequestStatus.ACCEPTED.getStatus ());
+            // required to be null for a void method
+            return null;
+        }).when(mockTransformer).transformDomainToJaxb(any());
 
-        EasyMock.replay(mockTransformer);
-        EasyMock.replay(mockClient);
-
-        this.individualRequestConsumer.processIndividualRequest(individualRequest, CONNECTION_TIME_OUT,
+        individualRequestConsumer.processIndividualRequest(individualRequest, CONNECTION_TIME_OUT,
                 RECEIVE_TIME_OUT);
 
-        EasyMock.verify(mockTransformer);
-        EasyMock.verify(mockClient);
-
-        Assert.assertTrue("Test finished successfully", true);
-
-    }
-
-    /**
-     * Test method for processing of individual request outage error.
-     */
-    @Test
-    public void processIndividualRequestOutage() {

```

Comment:
Any particular reason this test has been removed?

Reply:
Accidental. Reinstated